src/PKSim.UI/UIConstants.cs

```diff
@@ -39,6 +39,10 @@ public static class Size
          public static readonly int COMPOUND_VIEW_WIDTH = ScaleForScreenDPI(760);
 
          public static readonly int MODEL_PICTURE_HEIGHT = ScaleForScreenDPI(400);
+
+         public static readonly int SIMULATION_COMPOUND_CONFIGURATION_DEFAULT_HEIGHT = ScaleForScreenDPI(280);
```

Comment:
Just looks like it's working on my PC. Can you try on yours>

Reply:
will try it with the next nightly